tests/test_builtins.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 import numpy as np
 from scipy.ndimage import gaussian_filter
-from scipy.misc import ascent
+from scipy.datasets import ascent
```

Comment:
If I remember correctly this one behaves a bit differently and was leading to issues in the tests but if ci is green that's fine. The requirements need to be update though to scipy version with that import

Reply:
CI now fails with the old version. [See here](https://github.com/devitocodes/devito/actions/runs/12635281488/job/35204852799?pr=2510#step:11:32)